test/servinge2e/kourier/custom_route_test.go

```diff
@@ -159,7 +159,7 @@ func TestCustomOpenShiftRoute(t *testing.T) {
 		t.Fatalf("Error polling custom domain: %v", err)
 	}
 	const expectedResponse = "Hello World!"
-	if resp.StatusCode != 200 || strings.TrimSpace(string(resp.Body)) != expectedResponse {
+        if resp.StatusCode != 200 || !strings.Contains(string(resp.Body), expectedResponse){
```

Comment:
I don't get why this line is changed. The expected response is given and the trimmed response should simply match it.